frontend/src/pages/accounts/profile.page.tsx

```diff
@@ -221,7 +221,12 @@ const Profile: NextPage = () => {
       </>
     ) : (
       <>
-        <a className={styles["open-button"]} href="#mpp-choose-subdomain">
+        <a
+          className={styles["open-button"]}
+          href={
+            profile.has_premium ? "#mpp-choose-subdomain" : "/premium#pricing"
+          }
+        >
```

Comment:
I think you actually want to use `<Link>` here when referencing another page, to avoid full-page reloads:

```suggestion
        ( profile.has_premium
          ? <a
            className={styles["open-button"]}
            href="#mpp-choose-subdomain"
          >
          : <Link
            className={styles["open-button"]}
            href="/premium#pricing"
          >
        )
```

Reply:
Updated